roles/aodh/templates/etc/aodh/aodh.conf

```diff
@@ -35,17 +37,18 @@ connection = mysql+pymysql://aodh:{{ secrets.db_password }}@{{ endpoints.db }}/a
 signing_dir = /var/cache/aodh/api
 auth_url = {{ endpoints.keystone.url.admin }}
 auth_uri = {{ endpoints.keystone.url.internal }}
-project_domain_id = default
-user_domain_id = default
-project_name = service
-username = aodh
-password = {{ secrets.service_password }}
+admin_tenant_name = service
+admin_user = aodh
+admin_password = {{ secrets.service_password }}
 memcached_servers = {{ hostvars|ursula_memcache_hosts(groups, memcached.port) }}
+cafile = {{ aodh.cafile }}
+identity_uri = {{ endpoints.keystone.url.admin }}
```

Comment:
The aodh function verification is based on the net-mitaka env Niraj installed.
without these two parameter, aodh-api's https access cannot work. 

Please reference to mitaka version's aodh configuration document as below:
[Telemetry Alarming service configuration options](http://docs.openstack.org/mitaka/config-reference/telemetry/alarming_service_config_opts.html)

both cafile and identity_uri are mentioned, and not deprecated parameters.